articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md

```diff
@@ -72,11 +72,7 @@ On the `Shape` Class add the following Operation:
 
 - `Draw` with no return type.
 
-On the `Canvas` Class add the following Operation:
```

Comment:
This section is duplicated

![image](https://user-images.githubusercontent.com/5655789/212626220-d4926631-9254-41ed-ae6f-51b597fd9236.png)